protocol-designer/src/components/modals/FilePipettesModal/PipetteFields.tsx

```diff
@@ -198,30 +204,33 @@ export function PipetteFields(props: Props): JSX.Element {
           leftPipette={values.left.pipetteName}
           rightPipette={values.right.pipetteName}
         />
-        <div style={{ width: '13.8rem' }}>
-          <FormGroup
-            key="rightPipetteModel"
-            label={i18n.t('modal.pipette_fields.right_pipette')}
-            className={formStyles.stacked_row}
-            disabled={values.left.pipetteName === 'p1000_96'}
-          >
-            {renderPipetteSelect({
-              mount: 'right',
-              tabIndex: initialTabIndex + 3,
-              nameBlocklist: getIncompatiblePipetteNames(
-                values.left.pipetteName as PipetteName
-              ),
-            })}
-          </FormGroup>
-          <FormGroup
-            disabled={isEmpty(values.right.pipetteName)}
-            key={'rightTiprackModel'}
-            label={i18n.t('modal.pipette_fields.right_tiprack')}
-            className={formStyles.stacked_row}
-          >
-            {renderTiprackSelect({ mount: 'right', robotType })}
-          </FormGroup>
-        </div>
+        {has96Channel ? (
+          <div style={{ width: '13.8rem' }} />
```

Comment:
question

I guess this would be invisible (due to no height). Do we need this div?


```suggestion
          <div style={{ width: '13.8rem' }} />
```

Reply:
for some reason i thought i needed it but you're right i don't think it does anything. thanks! i'll clean it up